scalableminds/webknossos
# Improve warning-toasts regarding segmentation

### Mailable description of changes:
 - Segmentation-related warnings will be shown less frequent (e.g, not for skeleton tracings when there are multiple resolutions for segmentations anyway).

### URL of deployed dev instance (used for testing):
- `http://improvesegtoasts.webknossos.xyz`

### Steps to test:
- ensure no warnings are shown for datasets without segmentation
- for a dataset with only one segmentation-resolution:
  - no warning should be shown for resolution 0
  - ensure that a warning is shown regarding downsampled buckets, when being zoomed out a bit
  - ensure that a warning about unsupported volume is shown as soon as no segmentation is displayed anymore (zoom out further for that)
- for a dataset with multiple segmentation-resolutions:
  - no warning should be shown for resolution 0
  - ensure that a warning about unsupported volume data when being zoomed out a bit **in volume mode** (for view + skeleton, there shouldn't be a warning)

### Issues:
- fixes #2699 

------
